add-ons/ansible-service-broker/README.md

```diff
@@ -1,71 +1,84 @@
-# Ansible Service Broker Add-on
-
+Ansible Service Broker Addon
+======================
 
 This addon installs the [Ansible Service Broker](https://github.com/openshift/ansible-service-broker)
 
-## Prerequisites
+# Prerequisites
+
+This addon is designed for an RBAC enabled Openshift cluster of `>=3.7`. Minishift can be configured to
+deploy 3.7 with the following command:
+
+```
+$ minishift config set openshift-version v3.7.1
+```
 
 Minishift/CDK must be deployed with the Service Catalog enabled
 
-The Service Catalog can be deployed by adding the `--service-catalog` flag when starting Minishift. The ability to pass extra flags during startup can be enabled by setting the _MINISHIFT_ENABLE_EXPERIMENTAL_ environmental variable as follows:
+The Service Catalog can be deployed by adding the `--service-catalog` when starting Minishift. The ability to pass extra flags during startup can be enabled by setting the _MINISHIFT_ENABLE_EXPERIMENTAL_ environmental variable as follows:
 
 ```
 $ export MINISHIFT_ENABLE_EXPERIMENTAL=y
 ```
 
-Start Minishift with the `--service-catalog` flag:
+
+Start Minishift with the `--service-catalog` flag, and `--iso-url centos`
 
 ```
-$ minishift start --service-catalog
+$ minishift start --service-catalog --iso-url centos
 ```
 
-## Deploy the Ansible Service Broker
+If you are planning on building or pushing to the minishift registry, be sure
+to configure your shell using `eval $(minishift docker-env)`. This ensures the
```

Comment:
This line `This ensures the apb tooling can interface with your cluster's local registry` is bit confusing.